docs/sources/shared/wal-data-retention.md

```diff
@@ -0,0 +1,78 @@
+---
+aliases:
+- /docs/agent/shared/wal-data-retention/
+- /docs/grafana-cloud/agent/shared/wal-data-retention/
+- /docs/grafana-cloud/monitor-infrastructure/agent/shared/wal-data-retention/
+- /docs/grafana-cloud/monitor-infrastructure/integrations/agent/shared/wal-data-retention/
+canonical: https://grafana.com/docs/agent/latest/shared/wal-data-retention/
+description: Shared content, information about data retention in the WAL
+headless: true
+---
+
+The `prometheus.remote_write` component reuses the Prometheus Write-Ahead Log
+(WAL) implementation for resiliency against network outages and to avoid the
+prohibitively expensive in-memory buffering of all metric samples. The
```

Comment:
```suggestion
The `prometheus.remote_write` component uses a Write Ahead Log (WAL) prevent data loss during network outages. The
```

Looking at Robert's comment above... what if we simplified the initial sentence down to this?  Do we need to mention buffering to memory (RAM)?  I don't think it adds a lot... unless I'm missing a detail here.

Reply:
Ok, doing this